azure-pipelines.yml

```diff
@@ -72,7 +76,6 @@ jobs:
         name: py36_test_oldestdeps
 
       - linux: py38-devdeps
-        name: py38_test_devdeps
```

Comment:
why no name?

Reply:
it doesn't need a name because we don't reference it in the upload task later.

Reply:
But it won't have a nice name?

Reply:
the name is not used anywhere else iirc

Reply:
Is it not in the GUI?

Reply:
No, I don't think so the template generates that from the URL.and tox env iirc